tasks/collect-atlas-params/collect-atlas-params.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-atlas-params
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that collects Atlas API configuration options from the data file.
+  params:
+    - name: dataPath
+      type: string
+      description: |
+        Path to the JSON string of the merged data containing the Atlas config.
+  workspaces:
+    - name: data
+  results:
+    - name: bombasticApiUrl
+      type: string
+      description: |
+        URL of the bombastic API.
+    - name: ssoTokenUrl
+      type: string
+      description: |
+        URL of the SSO token issuer.
+    - name: secretName
+      type: string
+      description: |
+        The kubernetes secret to use to authenticate to bombastic.
+  steps:
+    - name: collect-atlas-params
+      image:
+        quay.io/konflux-ci/release-service-utils:d320c36f3d707cd5bfe55fe783f70236c06cc2e5
+      script: |
+        #!/usr/bin/env bash
+        set -x
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "ERROR: No valid data file was provided."
+            exit 1
+        fi
+
+        if ! jq -e '.atlasServer' "$DATA_FILE" > /dev/null; then
+            echo "ERROR: The JSON file does not contain the 'atlasServer' field."
+            exit 1
+        fi
+
+        atlasServer=$(jq -r '.atlasServer' "$DATA_FILE")
```

Comment:
Maybe better to just get it once and then check if it is null or empty string (I forget what `jq` does if missing) without the `-e` instead of calling the same query on line 48 and 53?

Reply:
refactored